modules/gapi/include/opencv2/gapi/util/util.hpp

```diff
@@ -84,6 +84,28 @@ namespace detail
     {
         static constexpr const std::size_t value = S;
     };
+
+    enum class PackageObjectTag {
```

Comment:
Potentially we can just derive from blank base class and use `std::is_base_of` when need to determine what type of object it is.

Reply:
Let's think about what if we extend this enum and how we are going to separate different entities in that case.

Reply:
Changed to be a blank class inheritance.